### PR TITLE
Fix createsession message bug

### LINKED ISSUE #200: [PE-D][Tester D] createsession success message bug
Success message missing a space between names of students

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: samuelmui8/ped#10

### LINKED ISSUE #201: [PE-D][Tester D] deletesession ux bug
after deleting a session, the following sessions are not flushed to the front. (eg if u delete session 3 and 4, session 5 does not become session 3), not very intuitive for users

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: samuelmui8/ped#11

### LINKED ISSUE #203: [PE-D][Tester A] Unit Test - Example given for the `createsession` command in the user guide does not work.
This is contained under the quick start section.

The command `createsession 3 n/John Doe ` throws this error to the user.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: AlagappanRa/ped#6

### PR DESCRIPTION
**Pull request by ldinghan**

### CODE REVIEW

**Comment by ldinghan** [comment]
Also provide fix for updatesessionremark for invalid session number

File: docs/UserGuide.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ The following describes what each panel contains:
 
     * `createconsult d/10/10/2023 tt/15:00 n/John Doe n/ Foo Bar` : Creates a consultation with the students `John Doe` and `Foo Bar`.
 
-    * `createsession 3 n/John Doe` : Creates a `No. 3` Session with the student `John Doe`.
+    * `createsession s/3 n/John Doe` : Creates a `No. 3` Session with the student `John Doe`.
```
Comment (Respirayson) [inline docs/UserGuide.md]: Perhaps creates a session with session number 3 would be clearer? I think someone talked about this before

**Review by Respirayson — APPROVED** [review]
Small nit but lgtm

File: docs/UserGuide.md (hunk 1 above)
Comment (ldinghan) [reply docs/UserGuide.md]: sounds good

**Pull request merged**